src/ui/atoms/input/example.tsx

```diff
@@ -31,6 +31,7 @@ export const InputExample: React.FC = () => (
               placeholder="Error"
               onChange={() => console.info('On input change')}
               variant="error"
+              label={<Label text="Error text" />}
```

Comment:
```suggestion
              label="Error text"
```

Не стоит заставлять пользователя импортировать компонент и дополнительно заворачивать обычный текст в компонент. Заверни в компонент внутри, самостоятельно.

Пусть компонент позволяет пользоваться с удобством.